### PR TITLE
Documentation of the "Anti External Links" module

### PR DESCRIPTION
**Pull request by FiloDiscord**

### CODE REVIEW

File: Wiki/zh-Tw.js [hunk 1]
```diff
@@ -80,7 +81,8 @@ exports.GENERAL = {
   MESSAGE: "訊息",
   USER: "使用者",
   SERVER: "伺服器",
-  NAME: "名稱"
+  NAME: "名稱",
+  URL: "URL"
```
Comment (xXLoloGameplayXx) [inline Wiki/zh-Tw.js:85]: A translation from Traditional Chinese is missing here.

File: Wiki/zh-Tw.js [hunk 2]
```diff
@@ -519,6 +521,48 @@ exports.MODULES = {
       SUBTITLE: "設定舉報頻道",
       SUGGESTION: "您設定的頻道需要是不公開且伺服器管理員可以看到的"
     }
+  },
+  ANTI_EXTERNAL_LINKS: {
```
Comment (xXLoloGameplayXx) [inline Wiki/zh-Tw.js:525]: A translation from Traditional Chinese is missing here.

P.S.: I have added some translations that have been recycled to build the documentation for this module.

**Review by xXLoloGameplayXx — CHANGES_REQUESTED** [review]
Several translations from Traditional Chinese are missing.

File: Wiki/zh-Tw.js (hunk 1 above)
Comment (kami-sqmf) [reply Wiki/zh-Tw.js:85]: I think it can be URL as well as it in Chinese

Comment (xXLoloGameplayXx) [reply Wiki/zh-Tw.js:85]: @kami-sqmf Ok 👍

File: Wiki/zh-Tw.js [hunk 3]
```diff
@@ -523,26 +523,31 @@ exports.MODULES = {
     }
   },
   ANTI_EXTERNAL_LINKS: {
-    DESCRIPTION: "This module will allow you to delete the message and warn members to send external links",
+    DESCRIPTION: "本模組將可以刪除指定的外部鏈接消息並警告發送者",
+    DISCORD_LINKS_BYPASS: "All Discord links will bypass this module",
```
Comment (xXLoloGameplayXx) [inline Wiki/zh-Tw.js:527]: A translation is missing here.

File: Wiki/zh-Tw.js [hunk 4]
```diff
@@ -523,26 +523,31 @@ exports.MODULES = {
...
-        TITLE: "Steps to add a path to the allowed list",
-        SUBTITLE: "Add a path to the allowed list",
-        INFORMATION: "You can allow all paths of a domain by replacing the path with an asterisk"
+        TITLE: "將路徑添加到允許清單的步驟",
+        SUBTITLE: "添加路徑到允許清單",
+        INFORMATION: "您可以透過用星號替換路徑，來指定所有的路徑"
       }
     },
+    DISALLOW_URL: {
+      TITLE: "Steps to remove a URL from the allowed list",
```
Comment (xXLoloGameplayXx) [inline Wiki/zh-Tw.js:548]: A translation is missing here.

File: Wiki/zh-Tw.js [hunk 5]
```diff
@@ -523,26 +523,31 @@ exports.MODULES = {
...
+      SUBTITLE: "Remove a URL from the allowed list"
```
Comment (xXLoloGameplayXx) [inline Wiki/zh-Tw.js:549]: A translation is missing here.

**Review by xXLoloGameplayXx — CHANGES_REQUESTED** [review]
There are three Traditional Chinese translations missing before you can Merge this PR.

**Review by xXLoloGameplayXx — APPROVED** [review]

**Pull request merged**